### ISSUE TITLE
ansible fails: cannot find /usr/bin/python

### ISSUE THREAD

**Issue by rstribrn** [vtorosyan/ansible-docker-vagrant-example#2]
...

**Comment by vtorosyan** [comment]
@rstribrn thnx for the issue. Fixed here 82a45966bc8f6a26c052510301fe2596a3ac72db

**Comment by c0bra** [comment]
@vtorosyan I just tried and got the same error:

```
==> default: Running provisioner: ansible...
    default: Running ansible-playbook...

PLAY [Setup environment to run the sample] *************************************

TASK [Gathering Facts] *********************************************************
fatal: [default]: FAILED! => {"changed": false, "failed": true, "module_stderr": "Shared connection to 127.0.0.1 closed.\r\n", "module_stdout": "/bin/sh: 1: /usr/bin/python: not found\r\n", "msg": "MODULE FAILURE", "rc": 0}
```
